### DISCUSSION TITLE
subselection not highlighted on same line

### DISCUSSION THREAD

**Discussion by benvds** [helix-editor/helix#5158] [Q&A] [answered]
I'm following the tutor.

Chapter 5.1 teaches about multiple cursors. Any selection on the first line is also represented on the second line.

At Chapter 5.2 it teaches about (sub)selections. When selecting a line with `x` and then `s` for search it adds another cursor but the 2nd subselection is not highlighted while the first is. I would expect the same visual behavior as with multiple cursors. 

Is this expected behavior?

See recording:

https://user-images.githubusercontent.com/95490/207706780-72c55cce-eed5-4baf-8200-ed8531a6da71.mp4

**Comment by gabydd** [comment, marked as answer]
The secondary selection should be highlighted so there is probably an issue with your theme, could you try a different theme and see if this issue still occurs?

**Reply by the-mikedavis** [reply]
Yeah I think the 'ui.cursorline(.primary)' background color for this theme is the same as secondary selections ('ui.selection'). Which theme is this? It could be updated so you could tell them apart.

**Comment by benvds** [comment]
Thx for the fast reply! The colorscheme is nightfox. When switching to another theme this issue seems to be resolved.
